File: Nível 5/1120 - Revisão de Contrato/main.lua
while true do
    input = io.read()
    digito, valorNegociado = input:match("(%S+)%s+(%S+)")
    digito, valorNegociado = tonumber(digito), tonumber(valorNegociado)

    if (digito == 0 and valorNegociado == 0) then
        break
    end

    removedNumber = string.gsub(valorNegociado, tostring(digito), "")

    if (removedNumber == "") then
        print(0)
    else
        print(tonumber(removedNumber))
    end
end

File: Nível 5/1120 - Revisão de Contrato/main.py
while (True):
    digito, valorNegociado = map(int, input().split(" "))

    if(digito == 0 and valorNegociado == 0): break

    removedNumber = str(valorNegociado).replace(str(digito), "")

    if(removedNumber == ""):
        print("0")
    else:
        print(int(removedNumber))